config/crd/bases/rhdh.redhat.com_orchestrators.yaml

```diff
@@ -465,6 +465,14 @@ spec:
                     type: string
                     default: orchestrator-gitops
                 type: object
+              networkPolicy:
+                description: ArgoCD contains the fields for the ArgoCD configuration
+                properties:
+                  existingRHDHNamespace:
```

Comment:
why not call it just `rhdhNamespace`?

Reply:
If `rhdhNamespace`, I was imagining the user would still think to configure it, even if the `rhdhOperator` field was enabled and configured. 
Now I think about it, it doesn't really matter if they do in that case. I can change it to `rhdhNamespace` if you would prefer? I'm fine with either :)

Reply:
I have updated the description for the `networkPolicy`. It had the wrong one, from bad copy/paste

Reply:
No, leave it as you suggested. Let's move on and merge this PR.